docs/content/Contribution guidelines/CONTRIBUTING.md

```diff
@@ -19,7 +19,7 @@ We can't wait to collaborate with you!
     - In go.mod it is indicated by the "go" directive.
     - In the .github/workflows yaml files it is indicated by "go-version"
     
-Check out our [Quickstart Guide](https://docs.kubestellar.io/release-v0.2.2/Getting-Started/quickstart/)
+Check out our [Quickstart Guide](/Getting-Started/quickstart/)
```

Comment:
This is broken for me when I test locally.
In http://localhost:8000/main/Contribution%20guidelines/CONTRIBUTING/ I see `<p>Check out our <a href="[/Getting-Started/quickstart/](view-source:http://localhost:8000/Getting-Started/quickstart/)">Quickstart Guide</a></p>`, and clicking on that my browser tries to go to http://localhost:8000/Getting-Started/quickstart/ and gets a 404 --- it should be going to http://localhost:8000/main/Getting-Started/quickstart/

Reply:
fixed